src/vm/excep.cpp

```diff
@@ -7164,6 +7164,13 @@ EXTERN_C void JIT_MemCpy_End();
 
 EXTERN_C void JIT_WriteBarrier_End();
 EXTERN_C void JIT_CheckedWriteBarrier_End();
+#else
+#ifdef FEATURE_PAL
+EXTERN_C void JIT_WriteBarrierGroup();
```

Comment:
I meant that you would rename those to use the ones you have added - the JIT_WriteBarrierGroup, JIT_WriteBarrierGroup_End etc. In your change, you have added your ones right next to the existing ones, so they represent the same range.